### PR TITLE
Removes Suicide

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
see title

## Why It's Good For The Game
cruft

## Changelog

:cl:
del: suicide verb
/:cl:

### CODE REVIEW

File: code/__DEFINES/flags.dm [hunk 1]
```diff
@@ -98,8 +98,6 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define HIDDEN_AREA (1<<6)
 /// If false, loading multiple maps with this area type will create multiple instances.
 #define UNIQUE_AREA (1<<7)
-/// If people are allowed to suicide in it. Mostly for OOC stuff like minigames
-#define BLOCK_SUICIDE (1<<8)
 /// Can the Xenobio management console transverse this area by default?
 #define XENOBIOLOGY_COMPATIBLE (1<<9)
```
Comment (thgvr) [inline code/__DEFINES/flags.dm]:
```suggestion
#define XENOBIOLOGY_COMPATIBLE (1<<8)
```

**Comment by BarteG44** [comment]
I am very much not a fan of these PRs

**Comment by thgvr** [comment]
There's not a whole lot suicides offer us honestly

**Comment by BarteG44** [comment]
What's the point of removing them tho?

**Comment by tiramisuapimancer** [comment]
suicide is lrp

**Comment by thgvr** [comment]
> What's the point of removing them tho?

We don't want them. We look like idiots telling people to not kill themselves if they have to go afk or leave a round while still having this feature around

**Comment by ritorizo** [comment]
Suicide has HRP uses I don't see why we should remove it.

**Comment by ritorizo** [comment]
Ok so my previous message was too "muh pr bad" I will elaborate why I think it's a bad idea to remove it.

Suicide is a gritty thing, but the shiptest community engage in gritty rp a lot. Murder, heavy trauma, torture and substance abuse are parts of the server which are there and people enjoy. Such things may not be enjoyed by everyone but are still part of the community. I can easily see someone kill themselves to avoid capture, out of madness or such.

However suicide has it is now is heavily lrp (like you can hold your breath or hug yourself to death lmao) which we might not want. Another issue is that suicide cause automatic DNR which make it  way less usable in rp.

TLDR; I would suggest removing the stupid suicide methods and make it not cause DNR instead.

**Comment by thgvr** [comment]
We don't need the suicide verb, you can still find other means

**Comment by MarkSuckerberg** [comment]
space is everywhere dawg, and it wants to kill you

**Comment by ritorizo** [comment]
Fair enough.

**Review by MarkSuckerberg — APPROVED** [review]
begone cruft
